worldmap.js

```diff
@@ -117,11 +116,95 @@ module.exports = function(RED) {
             }
             node.status({});
         });
-        sockets[this.path].on('connection', callback);
+        sockets[node.path].on('connection', callback);
+    }
+
+    var WorldMap = function(n) {
+        worldMap(this, n);
     }
     RED.nodes.registerType("worldmap",WorldMap);
 
 
+    function HTML(ui, config) {
+        var width = config.width;
+        if (width === 0) {
+            var group = RED.nodes.getNode(config.group);
+            if (group) {
+                width = group.config.width;
+            }
+        }
+        var height = config.height;
+        if (height === 0) {
+            height = 10;
+        }
+        var size = ui.getSizes();
+        var width = (size.sx +size.cx) *width -15;
```

Comment:
I think these calcs are slightly wrong... should be (size.sx +size.cx) * width - size.cx;  (one less space that number of units     [ sx - cx - sx - cx - sx ]   etc...  (but indeed may need a small fiddle factor to ensure iframe fits inside without scroll bars)

Reply:
Thank you. 
fixed expression.